cockatrice/src/phase.cpp

```diff
@@ -0,0 +1,29 @@
+#include "phase.h"
+
+Phase::Phase(const char *_name, QString _color, QString _soundFileName) : color(_color), soundFileName(_soundFileName)
+{
+    name = tr(_name);
+}
+
+Phase Phases::getPhase(int phase)
+{
+    if (0 <= phase && phase < Phases::phaseTypesCount) {
+        return phases[phase];
+    } else {
+        return unknownPhase;
+    }
+}
+
+const Phase Phases::unknownPhase(QT_TRANSLATE_NOOP("Phase", "Unknown Phase"), "black", "unknown_phase");
+const Phase Phases::phases[Phases::phaseTypesCount] = {
```

Comment:
I feel like these should be in the header file

Reply:
Either works, didn't bother moving them from the original pr.

Reply:
actually they are static and should be in the implementation file.